tomviz/ModuleVolume.cxx

```diff
@@ -191,6 +192,7 @@ void ModuleVolume::addToPanel(QWidget* panel)
   QCheckBox* maxIntensity = new QCheckBox("Maximum intensity");
   layout->addWidget(maxIntensity);
   QCheckBox* jittering = new QCheckBox("Jittering");
+  jittering->setChecked(true);
```

Comment:
You shouldn't need this line as line 202 sets it checked based upon the state of m_volumeMapper